js/webform_civicrm_payment.js

```diff
@@ -5,6 +5,13 @@ cj(function($) {
     setting = Drupal.settings.webform_civicrm,
     $processorFields = $('.civicrm-enabled[name$="civicrm_1_contribution_1_contribution_payment_processor_id]"]');
 
+  $(document).ajaxStart(function() {
+    $('#billing-payment-block').closest('form').block();
+  })
+  .ajaxStop(function() {
+    $('#billing-payment-block').closest('form').unblock();
+  });
+
```

Comment:
I don't think we should block the form for every use of ajax, just the one we care about. So I'd put this into the `loadBillingBlock` function.

Reply:
@colemanw I've got something similar for core, and you've hit the issue on the spot.  The problem with loadBillingBlock is it loads way too late on the form to be of any real use and you can still submit the form for a while before it freezes.

The ajaxStart/Stop functions can only be linked to `document`.  And if we want to limit it to only when the billing ajax request is going to be triggered we still want to do it as soon as the form loads at which point we don't know if it's going to have billing info on it.  By targetting #billing-payment-block we are making sure it doesn't target other forms but it will trigger for every ajax request on that form.

Reply:
Well, since this is using `cj` it will only attach to Civi's ajax requests not any initiated by Drupal, so... ok.